src/gpu/gpu_gemm_list.cpp

```diff
@@ -38,8 +38,8 @@ namespace {
 constexpr impl_list_item_t impl_list[] = {
         GPU_INSTANCE_INTEL_DEVMODE(intel::gemm::conv_t)
         GPU_INSTANCE_INTEL(intel::gemm::xe_hp_systolic_t)
-        GPU_INSTANCE_INTEL(intel::gemm::with_post_ops_t)
         GPU_INSTANCE_INTEL(intel::gemm::gen_t)
+        GPU_INSTANCE_INTEL(intel::gemm::with_post_ops_t)
```

Comment:
A comment above the list about the reasons why to keep `gen_t` above `with_post_ops_t` would be helpful.